examples/flutter_gallery/lib/demo/material/bottom_app_bar_demo.dart

```diff
@@ -160,28 +160,28 @@ class _BottomAppBarDemoState extends State<BottomAppBarDemo> {
         padding: const EdgeInsets.only(bottom: 88.0),
         children: <Widget>[
           const _Heading('FAB Shape'),
-
+          //
```

Comment:
??

Reply:
Without those comment empty lines are removed in list. Those comments act as separators. (It's a common formatter behaviour we could find in other langages)